src/components/Checkbox/Checkbox.stories.js

```diff
@@ -64,6 +72,42 @@ storiesOf('Checkbox', module)
         </div>
       </StyledCutout>
     ))
+  )
+  .add('menu', () =>
+    React.createElement(() => (
```

Comment:
Just out of curiosity, why are you using `React.createElement` here?

Reply:
Storybook has some strange bug and app crashes when you try to add a story with a component wrapped in `forwardRef` which uses hooks . this hack makes it work for some reason

Reply:
ok, cool!